### ISSUE TITLE
Allow asset map generation from GlimmerApp

### ISSUE THREAD

**Issue by erkarl** [glimmerjs/glimmer-application-pipeline#46]
I was trying to generate asset map so that my Service Worker knows which files to cache. It would be nice if GlimmerApp was similar to EmberApp:
```
  let app = new GlimmerApp(defaults, {
    fingerprint: {
      generateAssetMap: true
    },
  }
```

**Comment by rwjblue** [comment]
Completely agreed!  We are working on enhancements to bring things back in line with some of the capabilities of a normal ember-cli app...

**Comment by erkarl** [comment]
While trying to extend my GlimmerApp for a "temporary workaround" I noticed that `initializeOptions` isn't being called. Is this intended?

https://github.com/rickharrison/broccoli-asset-rev/blob/master/index.js#L7

**Comment by rwjblue** [comment]
This should be possible now (as of #81 and released in 0.6.3).
